.github/workflows/snyk-monitor.yaml

```diff
@@ -0,0 +1,46 @@
+name: Snyk Monitor
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-*'
```

Comment:
nit: all of the `v*` tags are protected and "reserved" for releases, so I guess that only the v* prefix is Ok.

Or you can alternatively connect this to the release WF.

Reply:
added v*